File: Softeer/Lv 1/softeer_근무 시간.py
import sys
input = sys.stdin.readline

total = 0

for _ in range(5):
    t1, t2 = map(str, input().split())
    t1 = t1.split(':')
    t2 = t2.split(':')

    hour = int(t2[0]) - int(t1[0])
    minute = int(t2[1]) - int(t1[1])

    if minute < 0:
        hour -= 1
        minute += 60

    total += (hour * 60 + minute)

print(total)

File: Softeer/Lv 1/softeer_근무 시간.c
#include <stdio.h>


int main(void)
{
	int h1, m1, h2, m2;
	int i; //반복문
	int hour, minute; //시간, 분
	int total=0; //전체 시간(분)

	for(i=0; i<5; i++)
	{
		scanf("%d:%d %d:%d", &h1, &m1, &h2, &m2);

		hour = h2 - h1;
		minute = m2 - m1;

		if(minute < 0)
		{
			hour -= 1;
			minute += 60;
		}

		total += hour * 60 + minute;

	}

	printf("%d", total);

	return 0;
}